app/models/goal.py

```diff
@@ -1,5 +1,22 @@
 from app import db
 
-
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    tasks = db.relationship('Task', backref='goal', lazy=True)
```

Comment:
Awesome hook up to the `Task` model! Good job remembering to add our `lazy` load parameter. ✨